src/client/app/helpers/trusted.snippet.helper.ts

```diff
@@ -18,6 +20,18 @@ class TrustedSnippet {
         }
     }
 
+    reloadTrustedSnippets(): void {
```

Comment:
Probably fine to only call once per startup (e.g., in app.ts)

Reply:
Added.